src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -223,6 +224,47 @@ public void testUpdateUserVerificationStatus() throws Exception {
         assertThat(returnValue.getProgramToken(), is(equalTo("prg-eedaf875-01f1-4524-8b94-d4936255af78")));
     }
 
+
+    @Test
+    public void testUpdateUsertaxVerificationStatus() throws Exception {
+        String functionality = "updateUsertaxVerificationStatus";
+        initMockServer(functionality);
+        HyperwalletUser hyperwalletUser = new HyperwalletUser();
+        hyperwalletUser.token("usr-b8e7ff1d-a3c6-45a0-ae0a-62b74580caca");
+        hyperwalletUser.taxVerificationStatus(TaxVerificationStatus.REQUIRED);
+
+        HyperwalletUser returnValue;
+        try {
+            returnValue = client.updateUser(hyperwalletUser);
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+
+        assertThat(returnValue.getToken(), is(equalTo("usr-b8e7ff1d-a3c6-45a0-ae0a-62b74580caca")));
+        assertThat(returnValue.getStatus(), is(equalTo(HyperwalletUser.Status.PRE_ACTIVATED)));
+        assertThat(returnValue.getTaxVerificationStatus(), is(equalTo(TaxVerificationStatus.VERIFIED)));
```

Comment:
If the taxVerificationStatus is being updated to REQUIRED why is the response returning VERIFIED? Shouldn't it be REQUIRED?

Reply:
I believe this test is testing the ability to update a user from Required -> Verified, the above step is the initialization of arguments before running the test